atd-vzv/src/views/map/InfoBox/InfoCard.js

```diff
@@ -17,56 +16,52 @@ const StyledTableRow = styled.tr`
   }
 `;
 
-export default class InfoCard extends PureComponent {
-  render() {
-    const { content } = this.props;
+const InfoCard = ({ content }) => {
```

Comment:
I did a quick search about PureComponents since it has been a while since I've thought of them. The bug fix is much more valuable than the small performance optimization that a PureComponent would give here so this is great. [This](https://blog.logrocket.com/what-are-react-pure-functional-components/) is the blog post that I found about it.